tools/flight_recorder/components/loader.py

```diff
@@ -39,11 +39,11 @@ def read_dir(prefix: str, folder: str) -> Dict[str, Dict[str, Any]]:
     gc.disable()
     details = {}
     t0 = time.time()
+    tb = time.time()
     for root, _, files in os.walk(folder):
         for f in files:
             ta = time.time()
             details[f] = read_dump(prefix, os.path.join(root, f))
             tb = time.time()
```

Comment:
Maybe we can just not calculate `tb` inside the loop. Probably rename it too.

```suggestion
    for root, _, files in os.walk(folder):
        for f in files:
            ta = time.time()
            details[f] = read_dump(prefix, os.path.join(root, f))
    tb = time.time()
```